src/application/tools/prepare-cluster/__tests__/prepare-cluster.test.ts

```
@@ -43,8 +43,8 @@ describe('prepare-cluster tool', () => {
         },
       });
 
-      const mockGet = mockContext.sessionService!.get;
-      mockGet.mockResolvedValue(session);
+      const sessionService = mockContext.sessionService as any;
+      sessionService.get.mockResolvedValue(session);
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)